### PR TITLE
Use IO extensions.

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [x] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Use Kotlin's IO extensions.

#### APK testing 

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt [hunk 1]
```diff
@@ -25,32 +22,13 @@ object LicenseFragmentHelper {
...
-                    "</head>",
-                    "<style>" + getLicenseStylesheet(context) + "</style></head>"
-                )
-            }
+                .replace("</head>", "<style>${getLicenseStylesheet(context)}</style></head>")
         } catch (e: IOException) {
-            throw IllegalArgumentException(
-                "Could not get license file: " + license.filename, e
-            )
+            throw IllegalArgumentException("Could not get license file: " + license.filename, e)
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt:30]:
```suggestion
            throw IllegalArgumentException("Could not get license file: ${license.filename}", e)
```

**Review by Stypox — CHANGES_REQUESTED** [review]
Nice, thanks

**Comment by TacoTheDank** [comment]
@Stypox Nice commit name :P

**Review by Stypox — APPROVED** [review]
I tested and it works fine. Thanks

> Nice commit name :P

Mmmh?

**Pull request merged**